Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -483,7 +485,7 @@
   - INSULT THE CLOWN
   - INSULT THE CREW
   - LIE
-  - MAKE FART NOISES
+  - MAKE DATED REFERENCES
```

Comment:
Borgs can't make fart noises easily so this is just QOL to make sure they can act on their funny law